src/main/flight/pid.c

```diff
@@ -344,6 +344,12 @@ static void pidApplyRateController(const pidProfile_t *pidProfile, pidState_t *p
         pidState->errorGyroIfLimit = ABS(pidState->errorGyroIf);
     }
 
+#ifdef USE_SERVOS
```

Comment:
It would be interesting to make all navigation/PID/IMU code being able to compile for fixed-point arithmetic. This way we might be able to squese navigation function into CJMCU as well.